ci.nix

```diff
@@ -70,6 +70,10 @@ mkShell {
   ETCD_BIN = "${etcd}/bin/etcd";
 
   shellHook = ''
+    ${pkgs.lib.optionalString (!nospdk) "echo 'SPDK version    :' $(echo $SPDK_PATH | sed 's/.*libspdk-//g')"}
```

Comment:
Should this be added to shell.nix as well?

Reply:
done